### PR TITLE
J24BTW-25: Back-End: API to GET List of all Inventory Items

### PR DESCRIPTION
**Pull request by timho1985**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 const app = express();
 const cors = require("cors");
+const inventoriesRouter = require("./routes/inventoriesRouter");
 
 require("dotenv").config()
 const { PORT } = process.env;
```
Comment (trupti-patil23) [inline server.js]: if reading port from config file failed in that case, plz add alternate port  using || .

File: server.js [hunk 2]
```diff
@@ -14,6 +15,8 @@ app.get("/", (req, res) => {
     res.send("Welcome to Instock server!");
```
Comment (trupti-patil23) [inline server.js]: if it is for testing connection, remove from 14-16 or else line no 15

File: server.js (hunk 1 above)
Comment (timho1985) [reply server.js]: Thank you. That is a good point

File: server.js (hunk 2 above)
Comment (timho1985) [reply server.js]: Sure

File: routes/.placeholder
Comment (trupti-patil23) [inline routes/.placeholder:1]: you can create warehousesRouter.js if you want , else please delete placeholders or empty files.

Comment (timho1985) [reply routes/.placeholder:1]: These files are been removed in my branch.

**Pull request merged**